docs/admin/pool_operations.md

```diff
@@ -585,24 +585,34 @@ tank     47 GB  0%   0%        0/32
 
 This returns a table of pool labels (or UUIDs if no label was specified)
 with the following information for each pool:
-- the total pool size
-- the percentage of used space (i.e., 100 * used space  / total space)
-- the imbalance percentage indicating whether data distribution across
+- The total pool size (NVMe or DATA tier, not including Metadata tier).
```

Comment:
In the SCM-only case (admittedly a corner case at this point), nothing has changed, has it? May be worth calling out here.

Reply:
can update in a follow-on, the behaviour for PMem has changed because only NVMe tier is displayed in the default mode, as you have said nothing changes for SCM-only (and there is no equivalent supported mode for MD-on-SSD)